sklearn/utils/tests/test_random.py

```diff
@@ -177,6 +182,30 @@ def test_random_choice_csc_errors():
                   class_probabilities, 1)
 
 
+@pytest.mark.parametrize("low,high,base",
+                         [(-1, 0, 10), (0, 2, np.exp(1)), (-1, 1, 2)])
+def test_loguniform(low, high, base):
```

Comment:
Because we only alias a scipy function, I'm not sure we really need tests for it.